### PR TITLE
Workspace

### PR DESCRIPTION
**Pull request by atenaiis**
- created NYTimes clone page we share it for a code review.

### CODE REVIEW

**Comment by sumancrest0001** [comment]
**STATUS: REQUIRED CHANGES
Good job @atenaiis and @KelynPNjeri** :clap:  :+1: 
It looks very similar to the original NYT site but we need to make a couple of changes to get approval for this project. So, let's fix the following issues.
[NOTE]changes with OPTIONAL are not mandatory but nice to have.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+<body>
+    <nav class="navbar">
+        <div class="icons">
+            <i class="fas fa-bars"></i> 
+            <i class="fas fa-search"></i>
+            <p>Space & Cosmos</p>
+        </div>
+        <div class="logo">
+            <img src="./assets/images/NYlogo.png" alt="Logo">
+        </div>
```
Comment (sumancrest0001) [inline index.html:20]: 1. NYT's logo at the navbar should be a link.

2. It seems that you forgot to a sticky footer at the bottom. Please have a look at the [original page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html).
3. [OPTIONAL] We can improve the alignment of horizontal like and NYT's logo at the bottom to give it the similar looks. See the differences.
Currently :point_down: 
Screenshot from 2020-05-22 08-04-44

In the original one :point_down: 
Screenshot from 2020-05-22 08-04-58

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+  padding: 5px;
+}
+
+.autor > a {
+  text-decoration: underline;
+  color: #121212;
+}
+
+.autor > a:hover {
+  text-decoration: none;
```
Comment (sumancrest0001) [inline assets/css/style.css:151]: [Optional] `text-decoration: none` is repeating multiple times. We can give it a class and add it as per our requirement.

**Comment by atenaiis** [comment]
Thank you for the code review, we worked on the observations you made.
* fix the logo to be a link
* added the sticky footer at the bottom 
* and made the improvement

**Comment by atenaiis** [comment]
hello,
we worked on the images for the Theory of inflation
on nav-bar  the Space and cosmos made it a link
The bottom now its change to links

**Pull request merged**